Makefile

```diff
@@ -191,3 +172,21 @@ endif
 md-lint: check-engine ## Lint markdown files
 	$(ECHO_PREFIX) printf "  %-12s ./...\n" "[MD LINT]"
 	$(CMD_PREFIX) $(CENGINE) run --rm -v $(CURDIR):/workdir --security-opt label=disable docker.io/davidanson/markdownlint-cli2:v0.12.1 > /dev/null
+
+##@ Validation
```

Comment:
This was a mistake, these are internal functions, we don't want to expose them IMO.

Reply:
So let's remove this line.

Reply:
Hmmm any reason we shouldn't have them documented though @leseb? Can you explain the motivation for not?

Reply:
Well, these are purely internal functions for the make code to operate correctly, like `_func()` in Python. They are not documented or exposed. Additionally, users are not supposed to use them, they could but it's meaningless, almost a distraction to the purpose of our Make file IMO.

Reply:
Sounds good guys, will do so in a followup commit